explainer.md

```diff
@@ -346,14 +346,101 @@ try {
+(somehow) marked as different from 'normal' WebAuthn credentials. It seems
+reasonable that an SPC credential should be able to be used as a login
+credential, if a Relying Party wished to do so. However whether a WebAuthn
+credential should be able to used in a payment authentication context is less
+clear.
+
+Allowing such a use would make it easier for an entity with existing WebAuthn
+credentials to use SPC, without requiring some explicit upgrade path. For
+example, an Account Provider may already have created WebAuthn credentials for their
+Customers to login, and might then want to use them for payment authentication
+as well.
+
+On the other hand, it would also open [a possible
+attack](#cross-origin-authentication-ceremony) on Relying Parties who do not
+perform sufficient verification of assertions that they receive. If a Relying
+Party fails to check the `clientDataJSON.type` or `clientDataJSON.origin` of
+the assertion, they might mistake a 3p-created SPC assertion for a login
+assertion.
+
+### Why use the Payment Request API?
+
+The proposed Secure Payment Confirmation API is built on top of the [Payment
+Request API] for authentication, defining SPC as a new 'payment method'. This
+was largely done for ease of implementation in Chrome, and for lack of any
+clearly better alternative.
+
+Using Payment Request does have some drawbacks:
+
+1. It has been argued that SPC does not make sense as a payment *method*, as
+   it conducts no actual payment. (However note that basic-card is also a
+   payment method and also conducts no actual payment - it only provides
+   instrument selection).
+
+1. There is a mismatch in API shape between the Payment Request API and SPC:
+     * Payment Request assumes that many payment methods can be passed in, but
+       SPC only makes sense on its own.
+     * Payment Request assumes that a Payment Handler window is still showing
+       after the `show()` promise resolves, and so has methods like `retry()`
+       and `complete()` on the `PaymentResponse`. For SPC, if the `show()`
+       promise resolves successfully there is nothing to retry or complete.
+
+Suggestions have been made to instead use `navigator.credentials.get()` as the
+entry point for SPC authentication. This seems reasonable but will require more
+cooperation with the WebAuthn WG.
+
+Related issues:
+* https://github.com/w3c/secure-payment-confirmation/issues/65
+* https://github.com/w3c/secure-payment-confirmation/issues/56
 
 ## Alternatives Considered
 
 ### Traditional WebAuthn
 
+WebAuthn level 2 (which allows `navigator.credentials.get` in a cross-origin
+iframe) does allow bringing FIDO-based authentication into online payments, by
+letting the Account Provider utilize it in a challenge iframe (instead of e.g.
+SMS OTP). However, as noted above, it is unable to support the following useful
+payments-specific abilities:
+
+1. Including transaction information in the signed assertion, which can be used
```

Comment:
s/transaction information/transaction information that was displayed to the user/

OR

s/transaction information/displayed transaction information/

Thoughts?